images/airflow/2.9.2/python/mwaa/config/sqs.py

```diff
@@ -124,13 +124,12 @@ def get_sqs_queue_name() -> str:
 
 def should_create_queue() -> bool:
     """
-    Determine whether the SQS queue should be created or not.
+    Determine whether the SQS queue should be created or not. Only used with CeleryExecutor.
```

Comment:
nit: I feel the addition to the comment here is not necessary. The implementation of this function is singularly concerned, it does not restrict based on executor type. It is the callers liberty to use this function as it sees fit. There could exist a scenario in the future where this is not used for just CeleryExecutor (for example, mocking a BatchExecutor). When / if this happens, this comment silently becomes a lie.